docs/content/en/docs/configuration/keys.md

```diff
@@ -353,6 +355,7 @@ Configure if HAProxy should maintain client requests to the same backend server.
 * `session-cookie-strategy`: the cookie strategy to use (insert, rewrite, prefix). `insert` is the default value if not declared.
 * `session-cookie-shared`: defines if the persistence cookie should be shared between all domains that uses this backend. Defaults to `false`. If `true` the `Set-Cookie` response will declare all the domains that shares this backend, indicating to the HTTP agent that all of them should use the same backend server.
 * `session-cookie-dynamic`: indicates whether or not dynamic cookie value will be used. With the default of `true`, a cookie value will be generated by HAProxy using a hash of the server IP address, TCP port, and dynamic cookie secret key. When `false`, the server name will be used as the cookie name. Note that setting this to `false` will have no impact if [use-resolver](#dns-resolvers) is set.
+* `session-cookie-keywords`: options to the `cookie` option like `nocache`, `preserve`, `httponly`. `indirect nocache httponly` is the default if not declared to be backwards compatible to earlier versions.
```

Comment:
add a reference to the default value only if `insert` strategy is used.

try also to rephrase the last sentence to something like "the default is ..." in order to separate the former list of keywords to the later one.

Reply:
Done